torch/jit/_monkeytype_config.py

```diff
@@ -72,6 +73,10 @@ def consolidate_types(self, qualified_name: str) -> Dict:
                     if inspect.getmodule(_type) == typing:
                         _type_to_string = str(_type)
                         _all_type += _type_to_string.replace('typing.', '') + ','
+                    elif inspect.getmodule(_type) == torch.nn.parameter:
+                        # If the type is a torch.nn.parameter module,
+                        # concatenate the module name and type name.
+                        _all_type += _type.__module__ + '.' + _type.__name__ + ','
```

Comment:
Nit:

Local temporary variables do not need to start with `_`, consider cleaning this up in a separate PR.